MyApp.Shared/GlobalRenderSettings.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace MyApp.Shared
+{
+    public static class GlobalRenderSettings
+    {
+        public static Microsoft.AspNetCore.Components.IComponentRenderMode? GlobalRenderMode { get; set; }
```

Comment:
How about the name `InteractiveRenderMode`?

The name `GlobalRenderMode` seems weird to me because if you really wanted to have a single rendermode everywhere, you don't need any of this - you'd just set it at the root level.

Reply:
Yes. I think that's the root of my confusion here.

Reply:
Name updated!